content/docs/studio/user-guide/experiments/explore-ml-experiments.md

```diff
@@ -102,6 +102,12 @@ There are two types of filters:
     On the other hand, the `Commits on branch` filter will also display branches
     [inside which the specified branch is nested](#nested-branches).
 
+    </admon>
+
+    <details>
+
+    ### More details on nested branches
```

Comment:
@yathomasi Is there any specific reason to change this structure? I feel that this filter a bit complex and needs the explanation upfront.

Reply:
It was more verbose and eye-catching for the user who is there for the second time or is not looking for that information. I think it's best practice to give detailed information inside the accordion with a proper title.